docs/source/index.rst

```diff
@@ -1,20 +1,204 @@
+common query language for multiple database backends.  The query language is:
+
+.. EDUCATIONAL: The pattern below is what you would call a definition list in restructuredtext.
+   The "terms" get special rendering in the readthedocs html file.
+
+Written in valid GraphQL syntax
+   Since it uses GraphQL syntax, the user get access to the entire GraphQL ecosystem,
+   including the typeahead capabilities and query validation capabilities of `GraphiQL
+   <https://github.com/graphql/graphiql>`__, user friendly error messages from the
+   reference GraphQL python implementation, and more.
+
+Directly compiled to the target database language
+   By compiling instead of interpreting the query language, the compiler highly improves query
+   performance and empowers the user with the ability to write deep and complex queries.
+   Furthermore, by using schema information from the target database, the compiler is able to
+   extensively validate queries, often more so than the DB-API, (e.g. :code:`pymssql`).
+
+Designed for cross-database querying
+   Since the query language always has the same semantics regardless of the underlying database,
+   we have been able to build a :doc:`Schema Stitching <advanced_features/schema_stitching>` system
+   that allows for seamless cross-database querying.
+
+Getting Started
+---------------
+
+Generating the necessary schema info
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+..  TODO: Encapsulate all schema info in a SchemaInfo class.
+
+To use the GraphQL compiler the first thing one needs to do is to generate the schema info from the
+underlying database as in the example below. Even though the example below generates schema info
+from an OrientDB database, it is meant as a generic schema info generation example.
+See the target database homepage for schema generation instructions.
+
+.. code:: python
+
+    from graphql_compiler import (
+        get_graphql_schema_from_orientdb_schema_data
+    )
+    from graphql_compiler.schema_generation.orientdb.utils import ORIENTDB_SCHEMA_RECORDS_QUERY
+
+    client = your_function_that_returns_an_orientdb_client()
+    schema_records = client.command(ORIENTDB_SCHEMA_RECORDS_QUERY)
+    schema_data = [record.oRecordData for record in schema_records]
+    schema, type_equivalence_hints = get_graphql_schema_from_orientdb_schema_data(schema_data)
+
+At the core of generated schema info is the GraphQL :code:`schema`. The database might be
+reflected in the :code:`schema` as follows:
+
+.. code::
+
+    type Animal {
+        name: String
+        out_Animal_LivesIn: [Continent]
+    }
+
+    type Continent {
+        name: String
+        in_AnimalLivesIn: [Animal]
+    }
+
+In the :code:`schema` above:
+
+-   :code:`Animal` represents a non-abstract vertex. For relational databases, we think of
+    tables as the non-abstract vertices.
+-   :code:`name` is a **property field** and :code:`out_Animal_LivesIn` is a **vertex field**,
```

Comment:
Remove the trailing comma here. The `which` refers to `vertex field`, and the comma is just confusing in that case. Bonus points if you split this into two bullet points, one explaining `property field` and one explaining `vertex field`.